### ISSUE TITLE
Using http://bin.rada.re/ with radare2 for Android?

### ISSUE THREAD

**Issue by Manouchehri** [radare/radare2-installer#8]
The [installer](https://play.google.com/store/apps/details?id=org.radare2.installer&hl=en) doesn't seem to be able to grab nightly binaries off http://bin.rada.re/, it's expecting `<base_url>/aarch64/unstable` or `<base_url>/aarch64/unstable`.

The builds on `http://radare.org/get/pkg/android/aarch64/` seem to be a few months out of date.

**Comment by radare** [comment]
The tarballs frm /pkg are built by me in local, those in jenkins cant be as trusted as those. Can you try to send a pullreq with this change? Or adding a new swttings option to use bin.rada.re instead of the /pkg ones if set?

**Comment by Manouchehri** [comment]
Hmm, I wonder if maybe just a termux package would be easier instead of a whole application just to install r2.

https://github.com/termux/termux-packages

**Comment by radare** [comment]
Yeah termux is great. But r2 doesnr compiles in there yet. Need to do a compiler profile for it. Thanks for pointing out that.

Anyway an app is good because it allows the user to install and use it from the ui. Not just the commandline
